invenio_theme_tugraz/ext.py

```diff
@@ -8,7 +8,7 @@
 
 """invenio module for TUGRAZ theme."""
 
-from flask_babelex import gettext as _
+# from flask_babelex import gettext as _
```

Comment:
```suggestion

```
If the file is not using this import it's better to remove it.

Reply:
i was not sure what your preferred approach is on such cases, because as i understand it correctly it comes from the cookiecutter template?

Reply:
yes - it does come with cookiecutter module, but we don't need it - you can remove it.